### PR TITLE
Updated Innersource Guidelines to better represent Github  as well as BitBucket

### PR DESCRIPTION
**Pull request by matthew-dexter-trimble**

### CODE REVIEW

File: content/innersource/codeowners-configuration.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ description: ""
 innersource: true
 hideToc: true
 ---
+For Github Code Owners is built in. See [About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in the Github documentation for configuration details.
```
Comment (coliff) [inline content/innersource/codeowners-configuration.md:8]: I know this is a Draft and not ready for review yet, but make sure to use the GitHub capitalization (capital H) everywhere. :-)

Comment (matthew-dexter-trimble) [reply content/innersource/codeowners-configuration.md]: Done

**Comment by kuhnboy** [comment]
Nice and comprehensvie!

**Review by kuhnboy — APPROVED** [review]

**Review by jeffdoolittle — APPROVED** [review]

**Comment by jeffdoolittle** [comment]
Thanks for making these changes, @matthew-dexter-trimble !

**Pull request merged**